tests/integration/conftest.py

```diff
@@ -54,7 +52,7 @@ def salt_call_cli(minion):
 
 @pytest.fixture(scope="session")
 def integration_test_config():
-    default_path = Path().parent.parent / "local" / "vcenter.conf"
+    default_path = Path(__file__).parent.parent.parent / "local" / "vcenter.conf"
```

Comment:
:+1:  That's probably more correct. It will be nice when we're actually setting things up *in* the test suite vs grabbing them.

Actually, it would be worth going ahead and adding a comment here:

> Long term the goal here is that our fixtures should connect to, and configure the vSphere the way we want. This pattern is not one that we really should follow, but at the moment it's the best option that we had.